### DISCUSSION TITLE
Feature Request: Ability to move multiple structure entries at once.

### DISCUSSION THREAD

**Discussion by Bshally** [craftcms/cms#8486] [Ideas]
Ordering large structures can become tedious when each entry must be moved one at a time. It would be more efficient to be able to check the box to select multiple entries then move them all at once by clicking the move icon of one of the selected items and dragging it into the desired position.

**Comment by foxtrotcz** [comment]
Agree, if you want to reorganize large structures, it would be much faster if you could move multiple entries at once.
Thanks.
